tools/run_tests/run_tests.py

```diff
@@ -1205,7 +1209,7 @@ def isclose(a, b, rel_tol=1e-09, abs_tol=0.0):
                            'node0.12', 'node4', 'node5', 'node6', 'node7', 'node8',
                            'electron1.3', 'electron1.6',
                            'coreclr',
-                           'cmake'],
+                           'cmake', 'cmake_vs2015', 'cmake_vs2017'],
```

Comment:
on windows, it's an alias for just 'cmake' or 'default' as VS2015 is our default windows compiler. So it's actually used but it's not invoked with the `cmake_vs2015` name. It might come handy once we change the default to VS2017 one day in the future.